version.sbt

```diff
@@ -1 +1,16 @@
-ThisBuild / version := "0.1.1-SNAPSHOT"
+/*
```

Comment:
The version here will get rewritten by the release plugin every time. So there is no reason to add it.

But not an issue at all.